.github/actions/launchable/setup/action.yml

```diff
@@ -88,8 +88,7 @@ runs:
       working-directory: ${{ inputs.srcdir }}
       run: |
         set -x
-        PATH=$PATH:$(python -msite --user-base)/bin
-        echo "PATH=$PATH" >> $GITHUB_ENV
+        echo "$(python -msite --user-base)/bin" >> $GITHUB_PATH
```

Comment:
To prevent this step from replacing all backslashes with forward slashes, `$GITHUB_PATH` is used. Based on my investigation, it seems to be that all paths are expressed as Unix path style when referring them in Bash.

https://github.com/ruby/ruby/actions/runs/9626581444/job/26576319914?pr=11043#step:10:271